xformers/factory/block_factory.py

```diff
@@ -130,12 +130,19 @@ def __init__(self, config: xFormerEncoderConfig, **kwargs):
             residual_scale=residual_scale,
         )
 
-        self.mha = build_multi_head_attention(config.multi_head_config)
-        self.feedforward = build_feedforward(asdict(config.feedforward_config))
+        mha = build_multi_head_attention(config.multi_head_config)
```

Comment:
we don't have to keep mha and feedforward as direct members, they were showing up in the model graph and when initializing, making it a little hard to see what was actually in the xformer model.